yurt/requirements.txt

```diff
@@ -1,3 +1,5 @@
 Django
 django_compressor
-psycopg2
\ No newline at end of file
+djangorestframework
+django-grappelli
```

Comment:
is the setup for `django-grappelli` already in the app (i.e. appropriate code in settings.py, etc), we just forgot to put it in `requirements.txt`?